test/Libraries/RevitNodesTests/Elements/Views/ViewTests.cs

```diff
@@ -36,6 +38,31 @@ public void ExportAsImage_BadArgs()
             Assert.Throws<System.ArgumentException>(()=>testView.ExportAsImage(""));
         }
 
+        [Test, TestModel(@".\Room_Properties.rvt")]
+        public void ElementsByCategory_FindRooms()
+        {
+            var testView = Document.Current.ActiveView;
+            var cat = Category.ById((int)Autodesk.Revit.DB.BuiltInCategory.OST_Rooms);
+            var elements = testView.ElementsByCategory(cat);
+            Assert.Greater(elements.Count, 1);
+        }
+
+        [Test, TestModel(@".\Empty.rvt")]
+        public void ElementsByCategory_EmptyList()
+        {
+            var testView = CreateTestView();
+            var cat = Category.ById((int)Autodesk.Revit.DB.BuiltInCategory.OST_Rooms);
+            var elements = testView.ElementsByCategory(cat);
+            Assert.Equals(elements.Count, 0);
```

Comment:
Here Assert.Equals should not be used for Assertions. Should replace it with Assert.AreEqual like
Assert.AreEqual(0, elements.Count);

Reply:
@ZiyunShang OK, just for my own clarification, is this a new requirement for tests? I based my added tests of existing.

Reply:
This Assert.Equals throws exception “Assert.Equals should not be used for assertions”.
It would be successful that use Assert.AreEqual instead.

Reply:
@ZiyunShang Correct, the node was moved to a different node and thus the other node removed and these tests no longer were valid.